Update README.md

agnosticdev opened the pull request:
Expanded and added a bit more clarity on the installation instruction for CMake on Linux.

<!-- What's in this pull request? -->
Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

<!-- If this pull request resolves any bugs in the Swift bug tracker, provide a link: -->
Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

<!--
Before merging this pull request, you must run the Swift continuous integration tests.
For information about triggering CI builds via @swift-ci, see:
https://github.com/apple/swift/blob/master/docs/ContinuousIntegration.md#swift-ci

Thank you for your contribution to Swift!
-->

agnosticdev committed:
Update README.md

Expanded and added a bit more clarity on the installation instruction for CMake on Linux.

CodaFi commented on README.md:
```diff
@@ -91,8 +91,13 @@ cloning over SSH may provide a better experience (which requires
 [CMake](https://cmake.org) is the core infrastructure used to configure builds of
-Swift and its companion projects; at least version 3.4.3 is required. Your
-favorite Linux distribution likely already has a CMake package you can install.
+Swift and its companion projects.  To get started using CMake, at least version 3.4.3 will be required. 
```
I think this line was fine the way it was before.

CodaFi commented on README.md:
```diff
@@ -91,8 +91,13 @@ cloning over SSH may provide a better experience (which requires
-favorite Linux distribution likely already has a CMake package you can install.
+Swift and its companion projects.  To get started using CMake, at least version 3.4.3 will be required. 
+Your favorite Linux distribution will most likely already have a CMake package you can install, and was included above in the installation instructions for Linux development dependencies, but in the event CMake is not installed on your Linux system it can be installed with the following command:
+
```
Could you move this below the macOS instructions and make it read like them too?

agnosticdev replied:
No problem at all. This has now been updated.

agnosticdev committed:
Update README.md

Reverted wording on the CMake description.  Added the Linux CMake instructions under the macOS instructions and reworded the Linux instructions to read like the macOS ones.

CodaFi commented on README.md:
```diff
@@ -90,19 +90,19 @@ cloning over SSH may provide a better experience (which requires
 
     export PATH=/Applications/CMake.app/Contents/bin:$PATH
+    
+On Linux, if the CMake package was not downloaded in the development dependencies you can download and install the CMake package on the terminal using the following command:
```
Nit: `on the terminal` is redundant

CodaFi commented on README.md:
````diff
@@ -90,19 +90,19 @@ cloning over SSH may provide a better experience (which requires
+
+``` 
+sudo apt-get install cmake
+```
````
In general, please wrap like the rest of the document

agnosticdev replied:
No problem.  This has been updated.

agnosticdev committed:
Update README.md

Removed redundant text and applied text wrapping.

gottesmm commented:
Can you smush these commits together before committing?

CodaFi commented:
@gottesmm I'll make sure to squash

@swift-ci please smoke test

gottesmm commented:
Thanks!

CodaFi commented:
Thanks @agnosticdev!  Welcome to the project.

⛵️

agnosticdev commented:
Thank you very much, @CodaFi !
